### DISCUSSION TITLE
How to configure Prometheus running on kubernetis to otel-collector -config file

### DISCUSSION THREAD

**Discussion by RSoodAMAT** [open-telemetry/opentelemetry-java#3337] [Q&A]
Hi , 

We are running Prometheus on Kubernetis cluster at 30090. It is a Node Port. 
When i tried configuring the same with my local otel -collector config -demo file . I get the below exception 

exporters:
  prometheus:
    endpoint: "k8s-m1:30090"
    namespace: traincenter
    const_labels:
      label1: value1
  logging:

Exception : 

otel-collector_1  | Error: cannot setup pipelines: cannot start builtExporters: listen tcp 10.200.7.39:30090: bind: cannot assign requested address
otel-collector_1  | 2021/06/21 13:43:57 application run finished with error: cannot setup pipelines: cannot start builtExporters: listen tcp 10.200.7.39:30090: bind: cannot assign requested address

What is the issue . 
Promethus is running on 9090 port with Node port being 30090.

**Comment by jkwatson** [comment]
I think this is a good question for the collector repository, rather than java. Please reach out on CNCF slack in the `#otel-collector` channel or the https://github.com/open-telemetry/opentelemetry-collector repository.
